File: src/main/java/fr/tikione/c2e/model/web/Paragraph.java
package fr.tikione.c2e.model.web;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Article paragraph.
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Paragraph {
    
    /** Raw text. */
    private String text;
}

File: app/src/main/java/fr/tikione/c2e/core/model/web/Paragraph.kt
package fr.tikione.c2e.core.model.web

/**
 * Article paragraph.
 */
class Paragraph {

    /** Raw text.  */
    var text: String? = null

    constructor() {
    }

    constructor(text: String?) {
        this.text = text
    }

    override fun toString(): String {
        return "Paragraph(text=$text)"
    }
}
